tools/eeprom-tool/.gitignore

```diff
@@ -0,0 +1,3 @@
+*.bin
```

Comment:
Why should these extensions be ignored?

Reply:
Because this is the extension used for files that are read / written from the memory and there is no use in adding them to the repo

Reply:
ok, but these files will be used next to the eeprom-tool executable, which will be somewhere in the build folder, not in the source tree.

Reply:
right. removed this .gitignore.